operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1581,3 +1581,79 @@ gateway:
       - secretName: gem-gateway-tls
         hosts:
           - gateway.gem.example.com
+
+metaMonitoring:
+  grafanaAgent:
+    # -- Controls whether to create PodLogs, MetricsInstance, LogsInstance, and GrafanaAgent CRs to scrape the
+    # ServiceMonitors of the chart and ship metrics and logs to the remote endpoints below.
+    # Note that you need to configure serviceMonitor in order to have some metrics available.
+    enabled: false
+
+    # -- Controls whether to install the Grafana Agent Operator and its CRDs.
+    # Note that helm will not install CRDs if this flag is enabled during an upgrade.
+    # In that case install the CRDs manually from https://github.com/grafana/agent/tree/main/production/operator/crds
+    installOperator: false
+
+    # -- Controls whether to scrape cadvisor, kubelet, and kube-state-metrics for metrics related to Mimir.
+    # The scraped metrics will be limited to ones concerning Mimir only: where pod
+    scrapeK8s: true
+
+    logs:
+      # -- Default destination for logs. The config here is translated to Promtail client
+      # configuration to write logs to this Loki-compatible remote. Optional.
+      remote:
+        url: ''
+
+        # -- Used to set X-Scope-OrgID header on requests. Usually not used in combination with basic authentication.
+        tenantId: ''
+        auth:
+          # -- Basic authentication username. Optional.
+          username: ''
+
+          # -- The value under key 'password' in this secret will be used as the basic authentication password. Optional.
```

Comment:
Would be nice to align with grafana.com and also to allow reusing the same secret for both logs and metrics, so I'd suggest instead of `passwordSecretName` -> `apiKeysSecretName` and instead of `password` -> `logsApiKey` and also for metrics: `password` -> `metricsApiKey`.

Reply:
I'll just do both